shared/common-adapters/text.native.js

```diff
@@ -20,6 +22,18 @@ class Text extends Component<void, Props, void> {
     openURL(this.props.onClickURL)
   }
 
+  _urlCopy = () => {
```

Comment:
I don't think this should be a part of our text class. why not have this be a part of the message stuff in chat?